_posts/2022-06-03-gaps-in-bitcoin-dev-funding.markdown

```diff
@@ -0,0 +1,30 @@
+
+## A funding bridge
+
+The big gap I see that isn't currently filled in the funding landscape is for people who can't quit their job or can only afford to work nights and weekends because of other commitments. Sure we see part-time grants, but building up one's credibility to get to that point takes investment. Funders aren't really taking a chance on people. The [residency](https://residency.chaincode.com/) filled this gap for some in the past. It was an almost-guarantee of finding FOSS funding. The residencies were never meant to scale and they are in hibernation anyways. Instead, we need a pot of cash to fund prospects. Let's call them bridge grants.
+
+Bridge grants are short-term grants, somewhere between 3 and 6 months, to wean a prospect off their job and give them a runway to show what they can do. They won't get rich from this funding, but it will be enough to avoid starvation and should inspire them to spend their time wisely establishing a track record that will attract a full-fledged grant.
+
+## A manager
+
+No strings attached funding provides the freedom to do meaningful work. In an industrialist society that no longer offers much respect to the individual worker, it reflects the trust of the grantee to choose the work that needs to be done. It's a thing of beauty.
+
+But, without strings, grantees don't go through the same growth of learning how to be professional -- AKA, how to show up when they don't feel like it. The kind of pain that comes with showing up to a bad job day after day because you made a promise to be there. No strings attached open-source funding doesn't help develop young people to be critical of management or understand when they need to quit a job. When a manager isn't parceling out tasks or providing (even poorly done) performance reviews, I believe it is harder for a green dev to get the feedback they need to grow as a developer and generally be consistently productive. Because FOSS doesn't signal whether you are performing well, I've seen people become demotivated and discouraged without a feedback structure that works (see wildly successful talent recruitment and retention of large tech companies).
+
+## A career
+
+Bitcoin open-source development isn't yet a career. Even in the US, where taking the road less traveled is culturally celebrated, health care is tied to employment. There is motivational value in career progression. While I don't believe in titles (see current descriptive title - Head of Special Projects), demonstrating career progression is helpful to mitigate the criticisms of friends and family who are already skeptical of the open-source path.
```

Comment:
I think the second sentence here needs to be developed more. And in general this section should be expanded upon.

Reply:
Career: "an occupation undertaken for a significant period of a person's life and with opportunities for progress."

There's opportunities to progress bitcoin :D

Reply:
i disagree